publication-event-handlers/src/test/java/no/unit/nva/publication/events/handlers/batch/BatchScanStartHandlerTest.java

```diff
@@ -0,0 +1,31 @@
+package no.unit.nva.publication.events.handlers.batch;
+
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+import java.io.IOException;
+import no.unit.nva.publication.events.bodies.ScanDatabaseRequest;
+import no.unit.nva.stubs.FakeContext;
+import no.unit.nva.stubs.FakeEventBridgeClient;
+import nva.commons.core.ioutils.IoUtils;
+import org.junit.jupiter.api.Test;
+
+class BatchScanStartHandlerTest {
+
+    private final FakeContext context = new FakeContext() {
+        @Override
```

Comment:
This code is duplicated, but I will fix it in commons and remove it from both places.